styles.css

```diff
@@ -83,14 +87,362 @@ header nav a {
   text-decoration: none;
 }
 
-nav {
+header nav {
   display: flex;
   align-items: center;
   justify-content: space-between;
   margin: 10px 0 0 0;
   margin-left: 20px;
 }
 
-nav img {
+header nav img {
   margin-right: 20px;
 }
+
+.works {
+  padding: 90px 20px 50px;
+  display: grid;
+  background: #f7f7f9;
+}
+
+.card-1 {
+  width: 290px;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 10px;
+}
+
+.card-1 h1 {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 33px;
+  margin: 16px 0 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding: 10px 0 0;
+  width: 250px;
+  align-items: baseline;
+  margin: 15px 0 10px;
+}
+
+.container li img {
+  margin: 0 5px;
+}
+
+.container h3 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #344563;
+  display: flex;
+  flex: none;
+  margin: 4px 0;
+}
+
+.container .counter1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.container .counter2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.languages {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  padding: 3px;
+  margin: 12px 0;
+  width: 200px;
+}
+
+.languages li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  color: #6070ff;
+  text-align: center;
+  height: 20px;
+  padding: 5px 10px;
+  border-radius: 11px;
+  background: #ebebff;
+}
+
+.card-1 p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+button {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```